Create ru_RU.lang for Core

SawFowl opened the pull request:
binniecore

SawFowl committed:
Create ru_RU.lang

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.rootBees.epitome.3=Профессиональный пчеловод
+binniecore.rootBees.epitome.4=Пчеловод мастер
+binniecore.rootBees.epitome.5=Бог пчеловодов
+binniecore.rootBees.epitome.6=Избранный Сенгир
```
Думаю, `Sengir's Chosen` Переводится как "Выбор Sengir'а".

Так как Sengir — один из основателей Forestry, его ник детранслитирировать не нужно.

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.rootBees.chromosome.lifespan.short=Продолжительность жизни
+binniecore.rootBees.chromosome.fertility.short=Плодовитость
+binniecore.rootBees.chromosome.temperature_tolerance.short=Температурный режим
+binniecore.rootBees.chromosome.never_sleeps.short=Cathemeral
```
Пропущено слово?

KorDum replied:
Оно означает "работу ночью" или "работу в подземельях"

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.rootButterflies.chromosome.fertility.short=Плодовитость
+binniecore.rootButterflies.chromosome.temperature_tolerance.short=Температурный режим
+binniecore.rootButterflies.chromosome.humidity_tolerance.short=Подходящая влажность
+binniecore.rootButterflies.chromosome.nocturnal.short=Cathemeral
```
Пропущено слово?

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.allele.fertility.high=Высокий
+binniecore.allele.fertility.maximum=Максимум
+
+binniecore.allele.none=None
```
Нужно посмотреть, как переведено это в Forestry. Возможно подойдет прочерк.

SawFowl replied:
В Forestry переводится как "Нет"

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.allele.growth.lightlevel=Уровень света
+
+binniecore.gui.breedingmessage.species=Обнаруженные виды
+binniecore.gui.breedingmessage.branch=Обнаруженные филиалы
```
Филиалы? Это же "ветви" спецфикации.

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+
+binniecore.item.container.capsule=Капсула
+binniecore.item.container.refractory=Капсула
+binniecore.item.container.can=Можно
```
"Капсула", а не "можно"

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.machine.tooltips.processing=Обработка
+
+binniecore.errors.task.cancelled.desc=Задача отменена
+binniecore.errors.task.cancelled.info=Отменена постройка ворот
```
Каких ворот? о_О

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.errors.task.paused.info=Пауза в постройке ворот
+
+binniecore.errors.unknown.name=Неизвестно
+binniecore.errors.unknown.desc=Эта ошибка не работает, сообщите об этом авторам мода
```
"Эта ошибка не работает" => "Неизвестная ошибка"

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.gui.database.breeder.stats=Статистика
+binniecore.gui.database.mutations.further=Дальнейшие мутации
+binniecore.gui.database.mutations.resultant=Результат мутации
+binniecore.gui.database.branch.branch=Филиал
```
Ветвь

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.gui.database.mutations.further=Дальнейшие мутации
+binniecore.gui.database.mutations.resultant=Результат мутации
+binniecore.gui.database.branch.branch=Филиал
+binniecore.gui.database.branch.name=%s Филиал
```
`Ветвь %s`

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.gui.database.branch.noDesc=Нет описания.
+binniecore.gui.database.mode.breeder=Селекционер
+binniecore.gui.database.discovered.discovered=Обнаруженные виды
+binniecore.gui.database.discovered.undiscovered=Не обнаруженные виды
```
"Необнаруженные" слитно

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.gui.genesis.title=Генезис
+
+binniecore.gui.fieldkit.title=Полевой комплект
+binniecore.gui.fieldkit.info=Полевой комплект анализирует пчел, деревья, цветы и бабочек. Все что требуется - это кусочек бумаги для заметок.
```
"Все, что потребуется, -"

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.allele.forestry.effectCreeper.desc=Сильно взрывается при приближении
+binniecore.allele.extrabees.effect.spawn_creeper.desc=Рандомно спавнит криперов
+binniecore.allele.extrabees.effect.blindness.desc=Ослепляет игроков
+binniecore.allele.forestry.effectDrunkard.desc=Причины игроков хмелеть
```
Что?

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.allele.extrabees.effect.gravity.desc=Притягивает объекты к своему улью
+binniecore.allele.extrabees.effect.bonemealsapling.desc=Удобряет саженцы
+binniecore.allele.forestry.effectHeroic.desc=Защищает свой улей от мобов
+binniecore.allele.extrabees.effect.hunger.desc=Заставляет игроков стать голодными
```
"Заставляет игроков голодать"

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.allele.forestry.effectHeroic.desc=Защищает свой улей от мобов
+binniecore.allele.extrabees.effect.hunger.desc=Заставляет игроков стать голодными
+binniecore.allele.extrabees.effect.lightning.desc=Стимулирует удары молнии
+binniecore.allele.extrabees.effect.meteor.desc=Заставляет метеориты падать с неба.
```
Точку в конце убрать

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.allele.forestry.effectMycophilic.desc=Распространяет грибы и мицелий
+binniecore.allele.forestry.effectNone.desc=Нет заметных эффектов
+binniecore.allele.forestry.effectMiasmic.desc=Отравляет игроков
+binniecore.allele.extrabees.effect.power.desc=Обеспечивает некоторую мощность для машин
```
"Заряжает машины" звучит лучше?

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.allele.extrabees.effect.power.desc=Обеспечивает некоторую мощность для машин
+binniecore.allele.forestry.effectRadioactive.desc=Уничтожает соседние блоки
+binniecore.allele.forestry.effectReanimation.desc=Восстанавливает умерших мобов
+binniecore.allele.forestry.effectRepulsion.desc=Естественно спавнит мобов, что бы избежать этого
```
Что?

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.allele.forestry.effectFestiveEaster.desc=Ничего не делает
+
+binniecore.gui.help.title=Help
+binniecore.gui.help.desc=Чтобы активировать подсказки, удерживайте %s и наведите указатель мыши на элементы управления..
```
Две точки в конце убрать

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+< Names for slots when using help tooltip >
+binniecore.gui.slot.slot=Слот
+binniecore.gui.slot.input=Входной слот
+binniecore.gui.slot.process=Технологический слот
```
"Слот обработки", он не технологический.

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.gui.slot.grain=Зерновой слот
+binniecore.gui.slot.tile=Слот инвентаря
+binniecore.gui.slot.caterpillar=Слот условного шва
+binniecore.gui.slot.polish=Польский слот
```
Какой "польский"? "Пыльца"!

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.gui.slot.circuit=Сетевой слот
+binniecore.gui.slot.grain=Зерновой слот
+binniecore.gui.slot.tile=Слот инвентаря
+binniecore.gui.slot.caterpillar=Слот условного шва
```
Это переводится как "Затирка"

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.gui.slot.polish=Польский слот
+binniecore.gui.slot.wood=Деревянный слот
+binniecore.gui.slot.acclimatiser=Слот для акклиматизатора
+binniecore.gui.slot.analyse=Анализ слота
```
"Слот анализа"

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.gui.slot.tile=Слот инвентаря
+binniecore.gui.slot.caterpillar=Слот условного шва
+binniecore.gui.slot.polish=Польский слот
+binniecore.gui.slot.wood=Деревянный слот
```
"Слот для дерева"

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.gui.slot.acclimatiser=Слот для акклиматизатора
+binniecore.gui.slot.analyse=Анализ слота
+binniecore.gui.slot.dye=Слот красителя
+binniecore.gui.slot.incubator=Слот для инкубатора
```
"Слот инкубации"

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.gui.slot.enzyme=Слот фермента
+binniecore.gui.slot.sequencervial=Слот для пробирки
+binniecore.gui.slot.catalyst=Слот катализатора
+binniecore.gui.slot.serum.input=Входная слот сыворотки
```
"Входящая сыворотка"

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.gui.slot.sequencervial=Слот для пробирки
+binniecore.gui.slot.catalyst=Слот катализатора
+binniecore.gui.slot.serum.input=Входная слот сыворотки
+binniecore.gui.slot.serum.output=Выходной слот сыворотки
```
"Выходящая сыворотка"

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.gui.slot.catalyst=Слот катализатора
+binniecore.gui.slot.serum.input=Входная слот сыворотки
+binniecore.gui.slot.serum.output=Выходной слот сыворотки
+binniecore.gui.slot.serum.active=Активный слот сыворотки
```
"Активная сыворотка"

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.gui.slot.serum.input=Входная слот сыворотки
+binniecore.gui.slot.serum.output=Выходной слот сыворотки
+binniecore.gui.slot.serum.active=Активный слот сыворотки
+binniecore.gui.slot.yeast=Слот дрожжей
```
"для"

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.gui.tank.amount=%s mB
+binniecore.gui.tank.percent.full=%s заполнено
+
+binniecore.gui.side.insert=Вставить сторону: %s
```
"Входящие стороны:"

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.gui.tank.percent.full=%s заполнено
+
+binniecore.gui.side.insert=Вставить сторону: %s
+binniecore.gui.side.extract=Извлечь сторону: %s
```
"Выходящие стороны:"

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.side.down=Низ
+binniecore.side.sides=Сторона
+binniecore.side.any=Произвольно
+binniecore.side.none=None
```
"Нет"

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+binniecore.side.any=Произвольно
+binniecore.side.none=None
+
+binniecore.key.hold.for.help.tooltips=Удерживайте для отображения справки
```
"Удерживайте КНОПКА для отображения..."

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,383 @@
+#Перевел Mr_Krab для проекта mc.beandex.ru
```
Это не нужно в файле локализации мода

KorDum commented:
I will oversee this translation.

SawFowl commented:
Переводил с помощью гугла. Местами перевод от яндекса. Могут быть ошибки. Если нашли, прошу поправить.

KorDum commented:
Спасибо за работу!
Я отписал в комментариях все, что нашел в этом файле, можно поправить прямо по ним.

SawFowl commented:
Я сегодня еще доработал перевод Forestry для 1.12.2, но вот скинуть его на официальный репозиторий не смог. Почему-то заблокирована там такая возможность. Языковые строки привел к одному виду с англоязычным файлом, переведен только текст, за основу взял оригинальный перевод и дополнил его.

KorDum commented:
По поводу перевода в Forestry: создайте в том репозитории issue с вопросом, почему так. Я, к сожалению, нахожусь в команде только этого мода :)

mezz commented:
Thank you both. Please mention me when these are ready and I will pull them.

KorDum commented:
@mezz Thank you! Since there are a lot of inaccuracies and machine translation in the text, it'll not be soon...

KorDum commented:
@Mr-Krab советую посмотреть на перевод для 1.7.10:
https://github.com/ForestryMC/Binnie/tree/master-MC1.7.10/src/main/resources/assets

SawFowl committed:
Update ru_RU.lang

Извиняюсь за задержку, произвожу обновления перевода по мере того как обновляю мод у себя на сервере.

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,384 @@
+binniecore.rootTrees.chromosome.height=Высота
+binniecore.rootTrees.chromosome.fertility=Плодовитость
+binniecore.rootTrees.chromosome.fruits=Фрукты
+binniecore.rootTrees.chromosome.yield=Уступание
```
Это переводится как "урожай"

KorDum commented on core/src/main/resources/assets/binniecore/lang/ru_RU.lang:
```diff
@@ -0,0 +1,384 @@
+binniecore.rootTrees.chromosome.height.short=Высота
+binniecore.rootTrees.chromosome.fertility.short=Плодовитость
+binniecore.rootTrees.chromosome.fruits.short=Фрукты
+binniecore.rootTrees.chromosome.yield.short=Уступание
```
"Урожай"

KorDum reviewed (changes requested):
Еще вот это заметил. И остался комментарий с "None" с предыдущего раза.

SawFowl commented:
Сейчас исправим.

SawFowl committed:
Update ru_RU.lang

SawFowl commented:
Обновил.

KorDum commented:
@Mr-Krab thanks for work!
@mezz all done :)

mezz commented:
Thank you!
